deploy-local.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker-compose up windbreaker rss-watcher
```

Comment:
Are the args `windbreaker rss-watcher` needed here?

Reply:
No, but just wanted to be explicit in case we later add in a `test` container